Algorithm/SINGON/11047.py

```diff
@@ -0,0 +1,16 @@
+import sys
+input = sys.stdin.readline
+N, K = map(int, input().split())
+cnt = 0
+A = []
+for _ in range(N):
+    X = int(input())
+    A.append(X)
+A.sort(reverse = True)
+for i in A:
```

Comment:
```suggestion
for i in A[::-1]:
```
문제에서 A는 오름차순으로 주어진다고 명시되어있어요~
또 다음과 같이 한번에 reverse 할 수 있답니다~

Reply:
아.. 이미 주어져 있었군요! 문제를 잘 읽어야겠네요! 감사합니다!